features/exacte_match.feature

```diff
@@ -0,0 +1,188 @@
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 7411CC   | 36         |            |                      | true        |
+  Dan bevat het resultaat de geen objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 7411CC   | 36         |            |                      |
+  | 7411CC   | 36         |            |                      |
+  
+Scenario: zoeken met combinatie van postcode en huisnummer waarbij exacteMatch kan worden toegepast kan niet i.c.m. fuzzy search
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 2391PH   | 1          |            |                      | false       |
+  Dan bevat het resultaat de volgende objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 2391PH   | 1          |            |                      |
+
+Scenario: zoeken met combinatie van postcode, huisnummer en evt. huisletter en/of huisnummertoevoeging en waarbij exacteMatch kan worden toegepast kan niet i.c.m. fuzzy search
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 7437SZ   | 2          |            | A                    | false       |
+  Dan bevat het resultaat geen objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  |          |            |            |                      |
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 1015DT   | 237        |            | A                    | false       |
+  Dan bevat het resultaat geen objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 1015DT   | 237        | C          | A                    |
+
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 3581KL   | 49         | A          |                      | false       |
+  Dan bevat het resultaat geen objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  |          |            |            |                      |
+
+  Gegeven de verzameling van objecten zoals beschreven bij Achtergrond
+  Als er wordt gezocht met
+  | postcode | huisnummer | huisletter | huisnummertoevoeging | exacteMatch |
+  | 1015DT   | 237        | A          |                      | false       |
+  Dan bevat het resultaat geen objecten
+  | postcode | huisnummer | huisletter | huisnummertoevoeging |
+  | 1015DT   | 237        | A          | V                    |
```

Comment:
Het geldt bij meer gevallen, maar ook bij deze. Door niet aan te geven wat er dus niet wordt teruggeven en de tabel niet in het blikveld zit, is het niet altijd duidelijk wat nou precies het effect is van de toepassing van exacte match.

Reply:
Tja, dat is het nadeel van deze opzet, maar het is het één of het ander, HC wil dit graag zo, ik laat even zo staan.

Reply:
> 
> 
> Tja, dat is het nadeel van deze opzet, maar het is het één of het ander, HC wil dit graag zo, ik laat even zo staan.

Dit begrijp ik niet. Kun je uitleggen waarom het niet mogelijk is om aan te geven wat er niet wordt teruggegeven als er tabellen wordt gebruikt? Ik neem aan dat je opmerking hierop slaat.

Reply:
Aanvankelijk stonden de data sets bij de scenario's. In de scenario's staat uiteraard wat het resultaat is en zo kon je (doordat de set bij het voorbeeld stond) direct zien wat er niet werd teruggegeven. Daarop werd aangegeven dat er wel erg veel informatie herhaald werd per scenario en werd aangegeven dat er een feature Achtergrond gebruikt kan worden waarin de data set staat. Die opzet heb ik nu gebruikt. Er is één data set bovenaan de featurebeschrijving toegevoegd, maar hierdoor verlies je bij het scenario het overzicht over wat er uit die data set NIET wordt geleverd omdat de data set mogelijk niet in beeld staat en doordat de data set meer data bevat dan relevant is voor dat scenario. Als we nu per scenario apart gaan benoemen wat er niet wordt geretourneerd, dan zijn we weer terug bij de initiële situatie. Ik stel voor om het zo te laten en niet nog meer tijd kwijt te zijn aan discussies over opmaak of wijzigingen daaraan, iedere opzet heeft voor- en nadelen. Laten we ernaar streven z.s.m. een release te maken!